README.md

```diff
@@ -62,6 +61,11 @@ First review [EIP-1](EIPS/eip-1.md). Then clone the repository and add your EIP
 | [658](https://github.com/ethereum/EIPs/pull/658)   | Embedding transaction status code in receipts                                                | Nick Johnson                               | Core       | Final    |
 | [706](EIPS/eip-706.md)                             | DEVp2p snappy compression                                                                    | Péter Szilágyi                             | Networking | Final    |
 
+# Active but Finalized EIPs (standards that have been adopted but can also be edited)
```

Comment:
"Active EIPs (standards that have been adopted but never meant to be completed)" should do.

Reply:
Thanks, I will make the appropriate change.